.readthedocs.yml

```diff
@@ -7,7 +7,7 @@
 version: 2
 
 build:
-  os: ubuntu-22.04
```

Comment:
This is not a GH action runner, but if this passes on RTD's runners then ok to update.